vaadin-radio-button-flow-demo/src/main/java/com/vaadin/flow/component/radiobutton/demo/RadioButtonGroupView.java

```diff
@@ -239,6 +247,81 @@ private void themeVariantsHorizontal() {
         addCard("Theme variants", "Direction", horizontal, vertical);
     }
 
+    private void dataViewRefreshItem() {
+        // begin-source-example
+        // source-example-heading: Refresh Items
+        RadioButtonGroup<Employee> radioButtonGroup = new RadioButtonGroup<>();
+        radioButtonGroup.setLabel("Assignable Employees: ");
+        Employee employee1 = new Employee("Employee One", null);
+        Employee employee2 = new Employee("Employee Two", null);
+        Employee employee3 = new Employee("Employee Three", null);
+        RadioButtonGroupListDataView<Employee> dataView = radioButtonGroup
+                .setItems(employee1, employee2, employee3);
+        radioButtonGroup.setValue(employee3);
+        radioButtonGroup.addThemeVariants(RadioGroupVariant.LUMO_VERTICAL);
+
+        Button updateButton = new Button("Update second employee's name!",
+                click -> {
+                    employee2.setTitle("Employee 2");
+                    dataView.refreshItem(employee2);
+                });
+        // end-source-example
+
+        addCard(DATA_VIEW, "Refresh Items", radioButtonGroup,
+                updateButton);
+    }
+
+    private void dataViewAddItem() {
+        // begin-source-example
+        // source-example-heading: Add Item
+        RadioButtonGroup<Employee> radioButtonGroup = new RadioButtonGroup<>();
+        radioButtonGroup.setLabel("Assignable Employees: ");
+        List<Employee> employeeList = new ArrayList<>();
+        employeeList.add(new Employee("Employee 1", null));
```

Comment:
The code is not using the buggy `Arrays.asList` since I used a `new ArrayList()` and added one employee to it to prevent that bug from happening, but, your proposed way to have a `List<Employee> employees = getEmployees();` is much more elegant with this demo context.

Reply:
Let's polish it a bit: `List<Employee> employeeList = getEmployeeList();`. This would let the users to see that exactly List is passed to setItems